tests/unit/backend/corpora/common/test_authorizer.py

```diff
@@ -1,23 +1,24 @@
+import json
 import os
 import unittest
 
 import requests
 from chalice import UnauthorizedError
 
 from backend.corpora.common.authorizer import assert_authorized
+from backend.corpora.common.utils.aws_secret import AwsSecret
 
-if not os.getenv("DEPLOYMENT_STAGE"):  # noqa
-    os.environ["DEPLOYMENT_STAGE"] = "test"  # noqa
 
-
-@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE") != "test", "DEPLOYMENT_STAGE not 'test'")
+@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE"), "DEPLOYMENT_STAGE not set")
 class TestAuthorizer(unittest.TestCase):
+    secret_name = "corpora/cicd/dev/auth0-secret"
```

Comment:
Move to inside `setUpClass` instead of floating?

Reply:
Is there a strong reason for that?

Reply:
Readability. It doesn't make sense to separate this class variable from the others and I'd wonder why.

Reply:
ok ok

Reply:
Can delete this line now that it has moved to line 19.